### DISCUSSION TITLE
Using JACK standalone with PipeWire?

### DISCUSSION THREAD

**Discussion by tristanbay** [DISTRHO/Cardinal#392] [Q&A]
So far, I haven't been able to get any sound out of the JACK standalone edition of Cardinal. I also have barely any experience in configuring JACK directly, through either GUI and/or the terminal. Also, I think that in vanilla VCV Rack, there's a way that you can just select the audio device(s)/outputs that you want to use from the output module. But in Cardinal, you seem to have to configure it externally. I'm using PipeWire, which generally has worked fine, and I have the pipewire-jack package installed on my Linux system. But I don't really know JACK terminal commands, and I don't currently have a GUI frontend for configuring it. Does anyone know how I can get sound with standalone JACK via PipeWire?

**Comment by dromer** [comment]
Use something like Catia or Qpwgraph to connect the application i/o.
